adding Bayesian ridge regression

t8ch opened the pull request:
from exp. neuroscientists at the MPI for Brain Research I have learned that they commonly use Bayesian regressors for decoding. Therefore, I thought it might be helpful to add them here as a reference.

t8ch committed:
adding Bayesian ridge regression

from exp. neuroscientists at the MPI for Brain Research I have learned that they commonly use Bayesian regressors for decoding. Therefore, I thought it might be helpful to add them here as a reference.

Commit:
Bayes added

jglaser2 commented:
Thanks so much for contributing, and sorry for the very long delay in getting to this! I agree it's useful to include this decoder. I'll merge the request and include the necessary parameters soon.

t8ch commented:
Hi,

you are welcome!
In the meantime I also implemented an early stopping option for the 
neural networks and added another DNN architecture. Let me know if 
that's interesting to you.

best

On 22.09.2017 19:57, jglaser2 wrote:
>
> Thanks so much for contributing, and sorry for the very long delay in 
> getting to this! I agree it's useful to include this decoder. I'll 
> merge the request and include the necessary parameters soon.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/KordingLab/Neural_Decoding/pull/2#issuecomment-331518012>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AbbSDYTi68IyeAedJ8XFUtqGsI8wkQDKks5sk_T1gaJpZM4PK1Cn>.
>

jglaser2 commented:
Cool. In the past I have also used different DNN architectures and early stopping. For now, I decided not to include them in the package to try to make all the functions easy to use (and so people wouldn't have to spend lots of time changing architectures and parameters). Maybe in the future I will include them as options. Thanks!
